robust-websocket.js

```diff
@@ -11,6 +11,8 @@
   }
 })(function(global, navigator) {
 
+  var WebSocket = global.WebSocket
```

Comment:
This doesn't make sense. either a) `WebSocket` is native and supported and never replaced, or b) it is polyfilled and we shouldn't close over and force the users to understand ordering of this library and the polyfill.

Reply:
The reason for this change is for another option: WebSocket has been overriden by browser extensions and users want to ensure that RobustWebSocket's implementation doesn't change from under them.